Science blog4 EN review

MikeMcC399 opened the pull request:
This PR corrects some minor grammar, language and whitespace issues.

It also harmonizes the thousands (comma) and decimal (full-stop) separators for numbers using English language conventions.

To create this PR I had to specify a compare string
`https://github.com/cuisines/cwa-website/compare/matriceria-add-science-blog4-en...MikeMcC399:science-blog4-en-review` as the cuisine fork was not offered by default. I don't know why this is the case, but I'm happy I found a way around it!

MikeMcC399 commented:
@matriceria / @dsarkar 
Could you please check this PR? I know it is in an unusual place!

Ein-Tim commented:
*Off Topic*

We should definitely try to stay in the https://github.com/corona-warn-app/cwa-website/ repo with things like this! 
I know, @MikeMcC399, you had no other option so this was the best you could do.

I think it would be good to give core contributors (outside from the community (like me or Mike)) privileges to be able to create branches in the https://github.com/corona-warn-app/cwa-website/ to avoid PRs which come from forks (where the community has to provide PRs to these forks in case they want to change something!).

But this is, in general, an uncommon case and does not matter right now.

MikeMcC399 commented:
There are issues with the Wikipedia links including incorrect and dead targets. I will add a commit to suggest fixes.

MikeMcC399 commented:
I have corrected the Wikipedia links. They were a little in disarray.

- In section 4.1 the iOS link pointed to iPhone. I changed that to point to iOS.
- The Android link pointed to a German Smartphone target. I changed that to an English Android target.
- In section 4.2 there were two dead links to iOS and Android which I fixed
- The "Bundesnotbremse" link was pointing from the English text "federal emergency brake" into the German Wikipedia article and the anchor was incorrect. So I corrected the anchor and linked the German text to the German word "Bundesnotbremse" since there was no equivalent English text on Wikipedia covering this topic specifically.

Also, because I was having issues using the Atom editor I switched to VScode which removed trailing spaces according to the conventions defined in [.vscode/settings.json](https://github.com/corona-warn-app/cwa-website/blob/master/.vscode/settings.json).

MikeMcC399 commented:
@Ein-Tim 
You raise here a complex organisational and technical issue regarding internal and external collaboration. I suggest this would be better placed in its own separate issue.

Ein-Tim commented:
@MikeMcC399 

Yes, you're right. I'll open an issue in the corona-warn-app/cwa-website repo later. Sorry for the off topic comment here. 😕

dsarkar commented:
Good morning @MikeMcC399 @Ein-Tim Thanks for the review. I will have a look later.

dsarkar commented:
@HelTob FYI

HelTob commented:
Good morning,

thank you very much for your contribution.
I really appreciate it.

Best regards,
Tobias

MikeMcC399 commented:
@HelTob 
Thank you for your feedback! I'm happy to assist!
